src/fhirtypes/common.ts

```diff
@@ -952,9 +952,12 @@ export function replaceReferences<T extends AssignmentRule | CaretValueRule>(
       }
     }
   } else if (value instanceof FshCode) {
+    // the version on a CodeSystem resource is not the same as the system's actual version out in the world.
+    // so, they don't need to match.
+    const baseSystem = value.system?.split('|')[0];
     const codeSystemMeta = fishForMetadataBestVersion(
```

Comment:
Since we're removing the version, I think this can just be `fishForMetadata` now, right @jafeltra?

Reply:
Yep! For simplicity, `fishForMetadata` should be sufficient. Looking at the implementation of `fishForMetadata`, if there is no `|` in the the value passed in, it actually doesn't make any difference and will only try to call `fishForMetadata`. But based on how we've used these functions, we've only been using `fishForBestMetadata` in the cases where we want the version match or the next best thing.